packages/components/Tabs/src/Tabs.tsx

```diff
@@ -82,10 +86,48 @@ export const Tabs = compose<TabsType>({
 
     const styleProps = useStyling(userProps, (override: string) => state[override] || userProps[override]);
 
+    if (Platform.OS == 'windows') {
```

Comment:
I'm wondering if this should just be a separate file completely - Tabs.windows.tsx. 

This is a lot of platform-specific code, and it makes this file a lot more complicated that it has to be. If there was just a line or two difference, it wouldn't matter. But this is a lot of Windows-specific logic. 

@ksiler @Saadnajmi @PPatBoyd Thoughts?

Reply:
Since we have Tabs.settings.**windows**.ts, I think it would make sense to split this into a .windows file

Reply:
Yeah, with us not splitting out the FocusZone, I think that makes sense.

Reply:
FWIW, I think we pushed most of the platform specifics to the windows specific slot props here, and have well documented where else we have platform.os checks